psalm.xml.dist

```diff
@@ -47,6 +49,19 @@
                 <referencedClass name="Interop\Container\Exception\ContainerException"/>
             </errorLevel>
         </InvalidThrow>
+        <DeprecatedMethod>
+            <errorLevel type="info">
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::addAbstractFactory"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::setService"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::setInvokableClass"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::setAlias"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::setShared"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::addInitializer"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::addDelegator"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::setFactory"/>
+                <referencedMethod name="Laminas\ServiceManager\AbstractPluginManager::mapLazyService"/>
```

Comment:
Should we get rid of these completely, or at least make them `private`? :thinking:

Reply:
Never mind: seen the deprecations below :)